### DISCUSSION TITLE
Gabapentin + MDMA

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#79] [Drug Interactions]
This does not appear to be a combination of concern. Though the paper does not directly address this combination, it does note that participants in the MDMA-AT study were allowed to continue taking gabapentin and did not note any adverse effects.

Christie, D., Yazar-Klosinski, B., Nosova, E., Kryskow, P., Siu, W., Lessor, D., & Argento, E. (2022). MDMA-assisted therapy is associated with a reduction in chronic pain among people with post-traumatic stress disorder. Frontiers in Psychiatry, 13. https://www.frontiersin.org/journals/psychiatry/articles/10.3389/fpsyt.2022.939302

**Comment by utaninja** [comment]
I don't see any reason for a pharmacological interaction for gabapentin and mdma. To note, this wouldn't go for the expansion as we are doing Pregabalin. But this is good for the general database to note. I'd be open to pushing this onto the database.

**Reply by zelixir25** [reply]
yea it just came up as i was searching for pregabalin info so i added it just so we have more information in general. i'm find pushin git to the db.

**Comment by utaninja** [comment]
Seems that this interaction has no reason to be of concern danger wise. Do you think there may be a concern of gabapentin masking the use of too much MDMA? As in the typical interaction of downers and stimulant concern.

Some sort of "Both can dull each other's effects, so if one wears off before the other it's possible to overdose due to the lack of counteraction" low risk plus decrease.

If not, this should be good to push this as Low Risk + No Synergy or Low Risk + Decrease

There does not seem to be reports of it dulling rolls, but typically downers are considered to do so.

**Comment by utaninja** [comment]
I have posted Pregabalin + MDMA in relevance to this combination - https://github.com/TripSit/drugs/issues/294
